feathr-example/bureau_bureau-balance.ipynb

```diff
@@ -0,0 +1,1224 @@
+    "    event_timestamp_column=\"TRAN_DATE\",\n",
+    "    timestamp_format=\"yyyy-MM-dd HH:mm:ss\"\n",
+    ")\n",
+    "\n",
+    "\n",
+    "# output would be in output_bureau.avro\n",
+    "client.get_offline_features(observation_settings=settings,\n",
+    "                            feature_query=feature_queries,\n",
+    "                            output_path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/output_bureau.avro\")\n",
+    "client.wait_job_to_finish(timeout_sec=7200)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the result and show the result\n",
+    "\n",
+    "Let's use the helper function `get_result_df` to download the result and view it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-09-02 11:40:43.272 | INFO     | feathr.spark_provider._synapse_submission:wait_for_completion:176 - Current Spark job status: success\n",
+      "2022-09-02 11:40:43.525 | INFO     | feathr.spark_provider._synapse_submission:download_file:427 - Beginning reading of results from abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/output_bureau.avro\n",
+      "Downloading result files: 100%|██████████| 201/201 [03:51<00:00,  1.15s/it]\n",
+      "2022-09-02 11:44:37.123 | INFO     | feathr.spark_provider._synapse_submission:download_file:456 - Finish downloading files from abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/output_bureau.avro to ../../../results/output_bureau.avro.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import shutil\n",
+    "def get_result_df(client: FeathrClient) -> pd.DataFrame:\n",
+    "    \"\"\"Download the job result dataset from cloud as a Pandas dataframe.\"\"\"\n",
+    "    res_url = client.get_job_result_uri(block=True, timeout_sec=600)\n",
+    "    tmp_dir = \"../output_bureau.avro\"\n",
+    "    shutil.rmtree(tmp_dir, ignore_errors=True)\n",
+    "    client.feathr_spark_launcher.download_result(result_path=res_url, local_folder=tmp_dir)\n",
+    "    dataframe_list = []\n",
+    "    # assuming the result are in avro format\n",
+    "    for file in glob.glob(os.path.join(tmp_dir, '*.avro')):\n",
+    "        dataframe_list.append(pdx.read_avro(file))\n",
+    "    vertical_concat_df = pd.concat(dataframe_list, axis=0)\n",
+    "    return vertical_concat_df\n",
+    "\n",
+    "df_res = get_result_df(client)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "          _c0 SK_ID_CURR SK_ID_BUREAU CREDIT_ACTIVE CREDIT_CURRENCY  \\\n",
```

Comment:
I'm not sure if this output is better served by a screenshot of the output in your instance.

Reply:
Cleaned up the outputs of the notebooks.